frontend/src/Judge/components/OtherMediaSubmission.js

```diff
@@ -1,5 +1,111 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Lightbox from 'react-images'
+import { Alert } from 'reactstrap'
 
-const OtherMediaSubmission = props => <div>Other Submission</div>
+import pdf from 'assets/pdf.svg'
+
+const PhotoContainer = styled.img`
+  height: 90%;
+  object-fit: scale-down;
+  width: 100%;
+`
+
+const PdfContainer = styled.img`
+  height: 100%;
+  object-fit: scale-down;
+  width: 45%;
+  background-color: #666666;
+  box-shadow: inset 2px 2px 1px rgba(0, 0, 0, 0.05);
+`
+
+const PdfText = styled.div`
+  position: absolute;
+  top: 80%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: black;
+`
+
+class OtherMediaSubmission extends Component {
+  static propTypes = {
+    path: PropTypes.string.isRequired
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      lightboxIsOpen: false
+    }
+  }
+
+  closeLightbox = () => {
+    this.setState({ lightboxIsOpen: false })
+  }
+
+  openLightbox = () => {
+    this.setState({ lightboxIsOpen: true })
+  }
+
+  render () {
+    const { path } = this.props
+
+    return (
+      <Fragment>
+        {path.endsWith('.pdf') ? (
```

Comment:
`endsWith` is not supported in IE, so unless we get it polyfilled by Babel, it'll need to change: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith